destroy-demo.sh

```diff
@@ -15,14 +15,14 @@ export ODOO_EXTRA_ADDONS=$DISTRO_PATH/odoo_config/addons
 export ODOO_CONFIG_PATH=$DISTRO_PATH/odoo_config/odoo_csv
 export ODOO_INITIALIZER_CONFIG_FILE_PATH=$DISTRO_PATH/odoo_config/config/initializer_config.json
 export ODOO_CONFIG_FILE_PATH=$DISTRO_PATH/odoo_config/config/odoo.conf
-export O3_FRONTEND_TAG=3.0.0-beta.13
-export O3_BACKEND_TAG=3.0.0-beta.13
+export O3_FRONTEND_TAG=
+export O3_BACKEND_TAG=
```

Comment:
That's what it should be except for the quick start script which gitpod uses. Except that for now I'm switching to using bleeding edge because of the changes I'm introducing. I will revert to released version alpha.7 when released.